.devops/templates/pr-target-branch.yml

```diff
@@ -0,0 +1,4 @@
+steps:
```

Comment:
Can the other pipeline files also be moved into .devops?

Reply:
probably only after we DUPLICATE them and then letting the build jobs get changed in the definition... but not for this PR.